evm/hyperliquid.js

```diff
@@ -169,26 +160,16 @@ async function main(processor, args, options) {
         const provider = getDefaultProvider(rpc);
         const wallet = new Wallet(options.privateKey, provider);
 
-        return processor(wallet, chain, options);
+        return processor(wallet, chain, args, options);
     });
 }
 
 async function switchHyperliquidBlockSize(options, useBigBlocks, chain) {
     const blockType = useBigBlocks ? 'big' : 'small';
-    options.args = [blockType];
     const rpc = chain.rpc;
     const provider = getDefaultProvider(rpc);
     const wallet = new Wallet(options.privateKey, provider);
-
-    try {
-        await updateBlockSize(wallet, chain, options);
-    } catch (error) {
-        throw error;
-    }
-}
-
-function shouldUseBigBlocks(key) {
-    return key === 'implementation' || key === 'interchainTokenFactoryImplementation';
+    await updateBlockSize(wallet, chain, [blockType]);
 }
 
 if (require.main === module) {
```

Comment:
Make the block size arg more restrictive by using `new Argument(...).choices(['big', 'small'])`
example [here](https://github.com/axelarnetwork/axelar-contract-deployments/pull/853/files#diff-834ca0a0151ef05d774cfd5ea8a7ecdec5e7302ee572c3a474d233242d60551dR463)